### PR TITLE
Implemented: logic to fetch receivers details(#85zrnfjuc)

### LINKED ISSUE #98: Show user full name on receive history.
## Current behavior
Currently we are only showing the user id not the users full name.

## Expected behavior
Need to show the user's full name which is more user friendly and readable.

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #98 

### Screenshots of Visual Changes before/after (If There Are Any)

Screenshot from 2023-02-21 12-43-29

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/receiving#contribution-guideline)

### CODE REVIEW

File: src/services/OrderService.ts [hunk 1]
```diff
@@ -32,9 +32,18 @@ const fetchPOHistory = async (payload: any): Promise<any> => {
   })
 }
 
+const getReceiversDetails = async (payload: any): Promise<any> => {
```
Comment (k2maan) [inline src/services/OrderService.ts:35]:
```suggestion
const getReceiverDetails = async (payload: any): Promise<any> => {
```

File: src/store/modules/order/OrderState.ts [hunk 2]
```diff
@@ -3,5 +3,6 @@ export default interface OrderState {
     list: any;
     total: number;
   },
+  receiversDetails: any;
```
Comment (k2maan) [inline src/store/modules/order/OrderState.ts:6]:
```suggestion
  receiverDetails: any;
```

**Review by k2maan — APPROVED** [review]

**Comment by k2maan** [comment]
We can have a discussion over the naming convention for variables with the word `receivers` in them.

File: src/store/modules/order/index.ts [hunk 3]
```diff
@@ -12,6 +12,7 @@ const OrderModule: Module<OrderState, RootState> = {
         list: [],
         total: 0
       },
+      receiversDetails: {},
```
Comment (adityasharma7) [inline src/store/modules/order/index.ts:15]: namesByUserLogin

File: src/store/modules/order/mutations.ts [hunk 4]
```diff
@@ -12,6 +12,11 @@ const mutations: MutationTree <OrderState> = {
   },
   [types.ORDER_CURRENT_PRODUCT_ADDED] (state, payload) {
     state.current.items.push(payload)
+  },
+  [types.ORDER_RECEIVERS_DETAIL_UPDATED] (state, receiversDetails) {
+    Object.keys(receiversDetails).map((receiversLoginId: any) => {
+      state.receiversDetails[receiversLoginId] = receiversDetails[receiversLoginId];
```
Comment (adityasharma7) [inline src/store/modules/order/mutations.ts:18]: Why not spread?

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**